bodhi/tests/utils.py

```diff
@@ -0,0 +1,31 @@
+# This program is free software; you can redistribute it and/or
```

Comment:
Let's add a Copyright claim above this (I've been trying to do this consistently across Bodhi, but a lot of modules still don't…). Something like this:

```
# -*- coding: utf-8 -*-
# Copyright © 2018 Red Hat, Inc.
#
# This file is part of Bodhi.
#
```

Reply:
Fixed in fixup commit